### PR TITLE
toggles could be uncheck

### PR DESCRIPTION
**Pull request by SamirTouira**
related to: #4789 

The buttons Premiere can be unchecked now

### CODE REVIEW

File: libs/movie/src/lib/movie/form/reviews/reviews.component.html [hunk 1]
```diff
@@ -34,7 +34,8 @@ <h3>Festival & Awards</h3>
 
             <h6 class="mat-caption">Premiere</h6>
             <div fxLayout="column" class="toggle-btn">
-              <mat-button-toggle-group [formControl]="prizesForm.get('premiere')" name="Premiere" aria-label="Premiere">
+              <mat-button-toggle-group [formControl]="prizesForm.get('premiere')" name="Premiere" aria-label="Premiere"
+                #group="matButtonToggleGroup" (change)="onChange()" (click)="onClick(group)">
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/form/reviews/reviews.component.html:38]: it is a bit redundant if you setting on every change the variable to true.
can you try something like that?
```typescript
,mat-button-toggle ... (change)="isChecked($event);

and in your ts file

isChecked(change: MatButtonToggleChange) {
if(this.form.get('premiereType').value === change.value) this.form.get('premiereType').reset();
}

Comment (fritzschoff) [reply libs/movie/src/lib/movie/form/reviews/reviews.component.html:38]: if so, then you also can delete the ref variable group and the click event listener.

File: libs/movie/src/lib/movie/form/reviews/reviews.component.ts [hunk 2]
```diff
@@ -11,10 +12,19 @@ import { DynamicTitleService } from '@blockframes/utils/dynamic-title/dynamic-ti
 })
 export class MovieFormReviewscComponent {
   form = this.shell.getForm('movie');
-
+  toggleChanged: boolean;
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/form/reviews/reviews.component.ts:15]: you can remove that

**Review by fritzschoff — CHANGES_REQUESTED** [review]

File: libs/movie/src/lib/movie/form/reviews/reviews.component.html (hunk 1 above)
Comment (SamirTouira) [reply libs/movie/src/lib/movie/form/reviews/reviews.component.html:38]: It strangly doesn't work. its toLabel('premiereType' isn't recognized... I don't really know how to get the value of these Premiere buttons after have tried many things

**Review by fritzschoff — APPROVED** [review]

**Pull request merged**